pkg/lib/config/authenticator.go

```diff
@@ -86,6 +88,57 @@ func (c *PasswordPolicyConfig) SetDefaults() {
 	}
 }
 
+var _ = Schema.Add("PasswordExpiryConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"force_change": { "$ref": "#/$defs/PasswordExpiryForceChangeConfig" }
+	}
+}
+`)
+
+var _ = Schema.Add("PasswordExpiryForceChangeConfig", `
+{
+	"type": "object",
+	"additionalProperties": false,
+	"properties": {
+		"enabled": { "type": "boolean" },
+		"since_last_update_in_days": { "$ref": "#/$defs/DurationDays" }
```

Comment:
Use DurationString. Refer to RateLimitConfig to see how the schema of RateLimitConfig is written in a way that `since_last_update_duration` is only required when `enabled` is true.